### ISSUE TITLE
typo in Examples - Interaction - Add/Delete Rows

### ISSUE THREAD

**Issue by freste** [olifolkerd/tabulator#3065]
**Website Page**
http://tabulator.info/examples/4.8#adddel

**Describe the issue**
"cleat" should be "clear":

   _**Tablulator**_ allows you to add new rows, delete existing rows and **_cleat_** all table data with ease.

Oh - also just noticed: "Tabluator" ! :-)

**Comment by olifolkerd** [comment]
Hey @freste 

Thanks for the headsup, i have fix it my end and will update the main site with the new release later in the month.

Cheers

Oli :)

**Comment by freste** [comment]
Hi!

Just a couple more minor documentation typos I noticed. I didn't want to
bother with github, hope this is ok.

http://tabulator.info/docs/4.9/download

In "Row Range" section:

all - All rows in the table reguardless of filters   => 'regardless'

(I don't think that's a British spelling?)



In CSV "Byte Order Mark":

interpereted across didfferent applications

==> interpreted across different applications


Thanks for your work, it's really quite good and fun to use! I just made a
small contribution. :-)
  -Fred
